tests/test-global-dependency.c

```diff
@@ -3,44 +3,44 @@
 // It also tests if the referances to the a and cc functions
 // will be preserved properly when moving.
 
-#include<stdio.h>
+/*
+get-ir:
+    status: success
+split:
+    status: success
+build:
+    status: success
+run:
+        stdout: name: a, exec: 9
```

Comment:
The indentation before `stdout` is definitely a bit too much! I'm also not sure if this formatting makes sense. I think it should be:

```
run:
  stdout:
    name: a, exec: 9
    name: c, exec: 10
```

Reply:
Maybe we should use `//` comments and then perhaps `clang-format` will leave them alone?

Reply:
It is fine to use `/*` I will just need not run clang-format -i on the tests.

Reply:
If we use `//` does clang-format not adjust things? It would be better to format tests if possible.

Reply:
What I mean is the after the initial formatting test's would not need to change. And clang format doesn't complain about comments but it might try to change them. That happens in both cases `//` and `/*`.

Reply:
Ah, if it happens with `//` and `/*` then it doesn't matter which we use.